package.json

```diff
@@ -44,6 +44,7 @@
     "eslint-plugin-react": "^7.19.0",
     "eslint-plugin-react-hooks": "^3.0.0",
     "ganache-cli": "^6.12.2",
+    "hardhat": "^2.4.0",
```

Comment:
Moving this to here as I realized the `protocol/node_modules/.bin/hardhat` was pointing to `protocol/packages/cloud-functions/node_modules/.bin/hardhat` which was causing an error when trying to run `hardhat x` commands from within `core`